DiceKit/ProbabilityMass.swift

```diff
@@ -8,28 +8,36 @@
 
 import Foundation
 
-public struct ProbabilityMass: ApproximatelyEquatable {
+// TODO: Find a proper place for this
+public struct ProbabilityMassConfig {
     
-    public typealias Outcome = FrequencyDistribution.Outcome
-    public typealias Probability = Double
+    public static let defaultProbabilityEqualityDelta: ProbabilityMass.Probability = 1e-16 // This is for 64-bit. What should 32-bit be?
+    public static var probabilityEqualityDelta: ProbabilityMass.Probability = defaultProbabilityEqualityDelta
+    
+}
+
+public struct ProbabilityMass<Outcome: FrequencyDistributionValueType>: ApproximatelyEquatable {
     
-    public static let defaultProbabilityEqualityDelta = 1e-16 // This is for 64-bit. What should 32-bit be?
-    public static var probabilityEqualityDelta = defaultProbabilityEqualityDelta
+    public typealias _FrequencyDistribution = FrequencyDistribution<Outcome>
+    public typealias _Outcome = Outcome
```

Comment:
Yeah, I think that it should be consistent, and the default is to not use `_` or make it the less used of the two.